plugins/arkworks/src/pairing.rs

```diff
@@ -118,14 +118,22 @@ pub mod test {
     #[test]
     fn bls12_381_has_valid_pairing_ratio() {
         let mut rng = OsRng;
-        assert_valid_pairing_ratio::<crate::bls12_381::Bls12_381>(rng.gen(), rng.gen(), rng.gen());
+        assert_valid_pairing_ratio::<crate::bls12_381::Bls12_381>(
+            rng.gen::<Standard, _>(),
+            rng.gen::<Standard, _>(),
+            rng.gen::<Standard, _>(),
```

Comment:
So this gives an error even though `Standard` implements `Default`. I think there may be a way around this eventually.

Reply:
It was before but for some reason there's no problem now, I changed it back to `rng.gen()`